elasticsearch/run.sh

```diff
@@ -162,13 +162,11 @@ seed_searchguard(){
         -tspass tspass \
         -nhnv \
         -icl ${DEBUG:+-dg}
-    
-    if [ $? -eq 0 ]; then
-      info "Seeded the searchguard ACL index"
-    else
-      error "Error seeding the searchguard ACL index"
-      exit 1
-    fi
+    do
+        warn "Error seeding the searchguard ACL index... retrying in 10 seconds"
+        sleep 10
+    done
+    info "Seeded the searchguard ACL index"
```

Comment:
I'm not going to approve this patch, but I'm not going to disapprove either.  If someone else is willing to approve, I have no objections.